src/komponenter/common/utloggingsvarsel/komponenter/ResizeHandler.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Dispatch, SetStateAction, useEffect } from 'react';
+import { verifyWindowObj } from '../../../../utils/Environment';
+
+export const BREAKPOINT = 768;
+
+export enum WindowType {
+    DESKTOP = 'DESKTOP',
+    MOBILE = 'MOBILE',
+}
+
+interface Props {
+    setWindowType: Dispatch<SetStateAction<WindowType>>;
+    windowType: WindowType;
+}
+
+const ResizeHandler = (props: Props) => {
+    const { windowType, setWindowType } = props;
+
+    useEffect(() => {
+        const resize = () => {
+            if (verifyWindowObj() && window.innerWidth > BREAKPOINT && windowType === WindowType.MOBILE) {
+                setWindowType(WindowType.DESKTOP);
+            } else if (verifyWindowObj() && window.innerWidth < BREAKPOINT && windowType === WindowType.DESKTOP) {
+                setWindowType(WindowType.MOBILE);
+            }
+        };
+
+        window.addEventListener('resize', () => resize());
+        return () => window.removeEventListener('resize', () => resize());
```

Comment:
```suggestion
        window.addEventListener('resize', resize);
        return () => window.removeEventListener('resize', resize);
```
...for at removeEventListener skal funke

Reply:
fikset på dette 👍